ablySpec/RealtimeClient.connection.swift

```diff
@@ -713,6 +713,44 @@ class RealtimeClientConnection: QuickSpec {
                 }
 
             }
+
+            // RTN10
+            context("serial") {
+
+                // RTN10c
+                it("should have last known connection serial from restored connection") {
+                    let client = ARTRealtime(options: AblyTests.commonAppSetup())
+                    defer { client.close() }
+                    let channel = client.channel("test")
+
+                    var lastSerial: Int64 = 0
+                    for _ in 1...5 {
+                        channel.publish("message", cb: { status in
+                            expect(status.state).to(equal(ARTState.Ok))
+                            lastSerial = client.connection().serial
+                        })
+                    }
+                    expect(lastSerial).toEventually(equal(4), timeout: testTimeout)
+
+                    let options = AblyTests.clientOptions(requestToken: true)
```

Comment:
Why are you using token auth here?  Not sure how that applies to this test when you have not used token auth for the first `client`